frontend/src/libs/components/link/link.tsx

```diff
@@ -1,15 +1,21 @@
 import { NavLink } from 'react-router-dom';
 
 import { type AppRoute } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import { type ValueOf } from '~/libs/types/types.js';
 
+import styles from './styles.module.scss';
+
 type Properties = {
   to: ValueOf<typeof AppRoute>;
+  className?: string | undefined;
```

Comment:
You can remove undefined here, because optional operator means that it could be undefined.

Reply:
fixed - [d7437aa](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/d7437aae577cad1a8cba8e73d2b767994f8260ba)

I am so ashamed of myself.